chat-plugins/badges.js

```diff
@@ -69,8 +69,10 @@ exports.commands = {
 			if (!Db('userBadges').has(userid)) return this.errorReply("This user doesn't have any badges.");
 			userBadges = Db('userBadges').get(userid);
 			selectedBadge = parts[2].trim();
+			targetUser = Users.getExact(userid);
 			userBadges = userBadges.filter(b => b !== selectedBadge);
 			Db('userBadges').set(toId(userid), userBadges);
+			if (Users.get(targetUser)) Users.get(userid).popup('|modal||html|<font color="red"><strong>ATTENTION!</strong></font><br /> Your ' + selectedBadge + ' badge have been taken by ' + Wisp.nameColor(toId(user), true));
```

Comment:
this basically says ``if (Users.get(Users.getExact(userid)))``

do you see how that's weird